pkg/controller/servicemesh/webhooks/server.go

```diff
@@ -0,0 +1,116 @@
+package webhooks
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+
+	"github.com/maistra/istio-operator/pkg/controller/common"
+	webhookcommon "github.com/maistra/istio-operator/pkg/controller/servicemesh/webhooks/common"
+	"github.com/maistra/istio-operator/pkg/controller/servicemesh/webhooks/mutation"
+	"github.com/maistra/istio-operator/pkg/controller/servicemesh/webhooks/validation"
+
+	maistrav1 "github.com/maistra/istio-operator/pkg/apis/maistra/v1"
+
+	arbeta1 "k8s.io/api/admissionregistration/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/types"
+)
+
+const componentName = "servicemesh-webhook-server"
+
+var log = logf.Log.WithName(componentName)
+
+// Add webhooks
+func Add(mgr manager.Manager) error {
+	log.Info("Setting up webhook server")
+	operatorNamespace := common.GetOperatorNamespace()
+	hookServer, err := webhook.NewServer(
+		componentName,
+		mgr,
+		webhook.ServerOptions{
+			Port:    11999,
+			CertDir: "/tmp/cert",
+			BootstrapOptions: &webhook.BootstrapOptions{
+				ValidatingWebhookConfigName: fmt.Sprintf("%s.servicemesh-resources.maistra.io", operatorNamespace),
+				MutatingWebhookConfigName:   fmt.Sprintf("%s.servicemesh-resources.maistra.io", operatorNamespace),
```

Comment:
I guess this was here before but I'm not sure I understand- what is this ConfigName about?

Reply:
That's the name of the ValidatingWebhookConfiguration object that the operator creates.